### ISSUE TITLE
Posts' image should have a max height 

### ISSUE THREAD

**Issue by tpkee** [aeharding/voyager#619]
Otherwise they may end up filling the whole screen space when unopened 
![IMG_0175](https://github.com/aeharding/voyager/assets/89136677/1d9fbe5c-0b28-4d0e-bf1b-8717fdeb8694)

**Comment by aeharding** [comment]
Images do have a max height, it's probably just longer than you're expecting.

**Comment by tpkee** [comment]
> Images do have a max height, it's probably just longer than you're expecting.

Yeah. Maybe it would be more pleasant to have a fixed height (something like 15rem), instead of relying on a min-max height, which makes the content look non uniform. We can use the [object-fit property ](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) to make the image resize to its box size so that it doesn't look bad.
